sp/src/game/server/ez2/ez2_player.cpp

```diff
@@ -724,6 +725,14 @@ Disposition_t CEZ2_Player::IRelationType( CBaseEntity *pTarget )
 {
 	Disposition_t base = BaseClass::IRelationType( pTarget );
 
+	if (pTarget && pTarget->Classify() == CLASS_COMBINE_HUSK)
+	{
+		// Bad Cop likes husks which are passive towards him
```

Comment:
Would this cause friendly fire lines to be spoken when Bad Cop kills a husk that is passive?

Reply:
It might do the default ones (e.g. "Uhhh... shit"), but not the soldier-specific ones because that response is based on the `npc_combine_s` classname.

Reply:
Won't there be a lot of situations where players deliberately stealth kill husks? (Taking them out before they have a chance to become non-passive)

Let me ask this: what's the purpose of this section of code to consider them friendly?

Reply:
> Won't there be a lot of situations where players deliberately stealth kill husks? (Taking them out before they have a chance to become non-passive)
I don't view this as a problem since players would need to give them an item or weapon to make them passive in the first place. If they wanted to kill a husk for reasons different from that of regular soldiers in E:Z2, they most likely would've done so before pacifying them.

> Let me ask this: what's the purpose of this section of code to consider them friendly?
It mainly engages the soldier friendly fire prevention code which prevents players from accidentally hitting them in combat, and also enables ally-specific responses.

Reply:
I think I misunderstood what "passive" means. Is that referring to husks that have already been befriended?

Reply:
Yep. I came up with the terminology before I came up with the recruiting/defending aspect 🥴